app/scripts/controllers/eventRegistrations.js

```diff
@@ -186,6 +186,13 @@ angular
       );
     };
 
+    $scope.isRegistrantReported = function(registrant) {
+      let t = $scope.registrations.find(r =>
+        r.registrants.some(rr => rr.id === registrant.id),
+      );
+      return t && t.reported;
+    };
```

Comment:
I like this better :)

I think it can be a little simpler:
```js
$scope.isRegistrantReported = function(registrant) {
  const registration = $scope.registrations.find(r => r.id === registrant.registrationId);
  return registration && registration.reported;
};
```